src/app/widget-builder/components/widgets/tips-widget/tips-widget-edit.component.html

```diff
@@ -0,0 +1,188 @@
+                                        <select formControlName="position">
+                                            <option *ngFor="let position of imagePositions" [value]="position.value">{{ ("LABEL_TIPS_EDIT_FORM_ELEMENTS_IMAGE_POSITION_OPTION_" + position.label |  uppercase) | translate }}</option>
+                                        </select>
+                                    </div>
+                                </div>
+                            </div>
+                            <div formGroupName="labels">
+                                <label>
+                                    <input type="checkbox" formControlName="enabled">
+                                    <span>{{ "LABEL_TIPS_EDIT_FORM_ELEMENTS_LABELS" | translate }}</span>
+                                </label>
+                                <div *ngIf="settings.items.labels.enabled">
+                                    <div formGroupName="limit_labels">
+                                        <label>
+                                            <input type="checkbox" formControlName="enabled">
+                                            <span>{{ "LABEL_TIPS_EDIT_FORM_ELEMENTS_LABELS_SHOW_ONLY_ALLOWED_LABELS" | translate }}</span>
+                                        </label>
+                                        <div *ngIf="settings.items.labels.limit_labels.enabled">
+                                            <label>{{ "LABEL_TIPS_EDIT_FORM_ELEMENTS_LABELS_ALLOWED_LABELS" | translate }}</label>
+                                            <input type="text" formControlName="labels">
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                            <div formGroupName="read_more">
+                                <label>
+                                    <input type="checkbox" formControlName="enabled">
+                                    <span>{{ "LABEL_TIPS_EDIT_FORM_ELEMENTS_READ_MORE_LINK" | translate }}</span>
+                                </label>
+                                <div *ngIf="settings.items.read_more.enabled">
+                                    <label>{{ "LABEL_TIPS_EDIT_FORM_ELEMENTS_READ_MORE_LINK_LABEL" | translate }}</label>
+                                    <input type="text" formControlName="label">
+                                </div>
+                            </div>
+                        </ng-template>
+                    </ngb-panel>
+                    <ngb-panel id="extra_parameters">
+                        <ng-template ngbPanelTitle>{{ "LABEL_TIPS_EDIT_FORM_PANEL_TITLE_EXTRA_PARAMETERS" | translate }}</ng-template>
+                        <ng-template ngbPanelContent>
+                            <!-- todo: Add querybuilder -->
+                            Querybuilder
```

Comment:
In eerste fase is querybuilder een gewoon tekstveld. Later komt er een link 'query builder' waarmee ze via interface dat tekstveld kunnen opvullen.